inclusive-design/idrc
# GH-461: indicate default color in color selectors (resolves #461)

## Description

Indicate default color in color selectors.

## Steps to test

1. Go to the Netlify CMS authoring interface at /admin/
2. Go to "Pages" collection, select any page, open "Header Background Color" selector, "Indigo 100 (Default)" is in the list.
3. Open other color selections, "(Default)" should be added to one color selection in the list.
